openedx/core/djangoapps/catalog/models.py

```diff
@@ -25,6 +27,14 @@ class CatalogIntegration(ConfigurationModel):
         )
     )
 
+    course_run_cache_ttl = models.PositiveIntegerField(
```

Comment:
@nasthagiri closing this and proceeding with the existing TTL (currently at 1 hour) sounds good.